crates/experimentation_client/src/lib.rs

```diff
@@ -68,11 +73,37 @@ impl Client {
                     self.client_config.tenant.to_string(),
                 )
                 .await;
+
                 match experiments {
                     Ok(experiments) => {
                         let mut exp_store = self.experiments.write().await;
-                        *exp_store = experiments;
-                        *start_date = Utc::now();
+                        for (exp_id, experiment) in experiments.into_iter() {
+                            if experiment.status.active() {
```

Comment:
can you check the main branch , we gave removed this logic 
do not do any change in this , just add the experiment_group logic